packages/cli/src/commands/bundle/getAssetDestPathAndroid.ts

```diff
@@ -4,12 +4,10 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
- * @flow strict
  */
 
 import path from 'path';
-import type {PackagerAsset} from './assetPathUtils';
+import {PackagerAsset} from './assetPathUtils';
 
 import assetPathUtils from './assetPathUtils';
```

Comment:
please consolidate imports :) 
```suggestion
import assetPathUtils, {PackagerAsset} from './assetPathUtils';
```